src/ray/rpc/client_call.h

```diff
@@ -47,7 +47,11 @@ using ClientCallback = std::function<void(const Status &status, const Reply &rep
 template <class Reply>
 class ClientCallImpl : public ClientCall {
  public:
-  void OnReplyReceived() override { callback_(GrpcStatusToRayStatus(status_), reply_); }
+  void OnReplyReceived() override {
+    if (callback_ != nullptr) {
```

Comment:
when would the `callback_` be null?

Reply:
It's user input param, user can enter null